### DISCUSSION TITLE
Set Title, Author or Producer when converting Images to PDF

### DISCUSSION THREAD

**Discussion by tomasr78** [dlemstra/Magick.NET#896] [Help]
Is it possible to set Title, Author, or Producer in PDF when converting Images to PDF?

**Comment by dlemstra** [comment]
This is currently not supported but this will become available in the next release with a new class called `PdfWriteDefines`.

**Reply by tomasr78** [reply]
Just would like to know maybe PdfWriteDefines is available already and how to use it?

**Reply by dlemstra** [reply]
You could use it like this:

```C#
using (var image = new MagickImage("input.jpg"))
{
    image.Write("output.pdf", new PdfWriteDefines
    {
        Author = "magick",
    });
}
```
